CONTRIBUTING.md

```diff
@@ -83,13 +83,37 @@ Please follow the below conventions when contributing to this project.
 * Each .NET project should have its corresponding test project
 * Each class should have its corresponding test class
 * Each internal and public function in the class should be tested in the unit test class
-* Follow the below naming conventions
+* Follow the below naming conventions:
   * Test Project: *{project name}.(Unit/Functional)Tests* selecting the appropriate type of tests found in the project
   * Test Class: *{class name}Tests.cs*
   * Test methods:
     * (Data)TestMethod: `{method name}_{what is being tested}_{expected outcome}`
     * DataRow (display name): Short description that clearly differentiate between the DataRows
 
+## Code Review Process
+
+### Before Creating a Pull Request
+* The changes cannot break any existing functional/unit tests that are part of the central repository.
+  * This includes all tests, even those not associated with the given feature area.
+* Code submitted must have [basic unit test coverage](https://github.com/Azure/template-analyzer/blob/development/CONTRIBUTING.md#tests), and have all the unit tests pass. Functional tests are encouraged.
+* Code submitted must have all the built-in rule tests pass. If applicable, code submitted should add [built-in rule test coverage](https://github.com/Azure/template-analyzer/blob/development/CONTRIBUTING.md#contributing-analyzer-rules).
+* Code should be commented.
+* Code should be fully code reviewed.
+* Code should be able to merge without any conflicts into the `development` branch.
+* Code should pass all relevant static checks and coding guidelines.
+* All build warnings and code analysis warnings should be fixed prior to submission.
+
+### Creating a Pull Request
+The Pull Request Template for this repo can be found [here](https://github.com/Azure/template-analyzer/blob/development/.github/pull_request_template.md).
+* Target the `development` branch for your Pull Request.
+  * The `main` branch is used for library releases. A release is created off of the `development` branch.
```

Comment:
I don't think this is true.  We should merge to main immediately before releasing, and create the release off the main branch.

Reply:
I think the simplest solution is to simply remove the point.  We only need to specify targeting development - what other branches are for doesn't need to be explained here. :)